### PR TITLE
Ad 417 ad 399

### PR DESCRIPTION
**Pull request by alexAlice**

### CODE REVIEW

File: packages/donations-app/app/index.html [hunk 1]
```diff
@@ -27,22 +27,44 @@
...
+          r=o.createElement('script');r.async=1;
+          r.src=t+h._hjSettings.hjid+j+h._hjSettings.hjsv;
+          a.appendChild(r);
+        }
     })(window,document,'https://static.hotjar.com/c/hotjar-','.js?sv=');
   </script>
 
+  <script>
+    (function () {
+      var timer = setInterval(function () {
```
Comment (jakub-wojciechowski) [inline packages/donations-app/app/index.html]: What do you think about Mutation Observe (https://stackoverflow.com/questions/3219758/detect-changes-in-the-dom), is it worth giving a try?

File: packages/donations-app/app/components/global/storyCard.html [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12 story-card__container">
   <div class="story-card">
     <!-- story card image -->
-    <img class="story-card__image" src="{{ story.img }}">
+    <img class="story-card__image lazy" data-src="{{ story.img }}">
```
Comment (jakub-wojciechowski) [inline packages/donations-app/app/components/global/storyCard.html]: Shouldn't we add data-src in other places? Is data-src compatible with ng-src that we use e.g. in the project splash?

Comment (alexAlice) [reply packages/donations-app/app/components/global/storyCard.html]: Discussed on call

File: packages/donations-app/app/index.html (hunk 1 above)
Comment (alexAlice) [reply packages/donations-app/app/index.html]: I have added a task on JIRA to check it later

**Pull request merged**